apps/sam-design-system-site/src/sds-styles.scss

```diff
@@ -2,11 +2,8 @@
 
 //global styles below
 
-$theme-image-path: '~/@gsa-sam/sam-styles/src/img';
-$theme-font-path: '~/@gsa-sam/sam-styles/src/fonts';
+$theme-image-path: '~@gsa-sam/sam-styles/src/img';
+$theme-font-path: '~@gsa-sam/sam-styles/src/fonts';
 
-// @import '~@gsa-sam/sam-styles/src/stylesheets/sam';
```

Comment:
Make sure it builds and works fine in angular 12 version

Reply:
Yeah this branch is off the angular-12 branch. The screenshot is from the federalist build.